index.js

```diff
@@ -1687,7 +1742,58 @@
   //. //
   //. //   The value at position 1 is not a member of ‘{ x :: FiniteNumber, y :: FiniteNumber }’.
   //. ```
-  function RecordType(fields) {
+  //.
+  //. ##### Extending record types
+  //.
+  //. By using the `parent` argument of the RecordType constructor, one can
+  //. define record types which must contain all the same properties of
+  //. another. For example, one could define `Poinst3D` as a refinement of
+  //. `Point` from the previous example:
+  //.
+  //. ```js
+  //. //    Point3D :: Type
+  //. const Point3D = $.RecordType(Point, {z: $.FiniteNumber});
+  //.
+  //. //    a :: Point
+  //. const a = {x: 0, y: 0};
+  //.
+  //. //    b :: Point, Point3D
+  //. const b = {x: 3, y: 4, z: 5};
+  //.
+  //. Point.validate(a)   // => Right(a)
+  //. Point3D.validate(a) // => Left({propPath: ['z'], value: undefined})
+  //.
+  //. Point.validate(b)   // => Right(b)
+  //. Point3D.validate(b) // => Right(b)
+  //.
+  //. dist(a, b);
+  //. // => 5
+  //. ```
+  //.
+  //. The `b` value is a member of `Point3D` as well as `Point`, allowing one
+  //. to call `dist` on it. By default record types allow additional
```

Comment:
I suggest replacing _allowing one to call `dist` on it_ with _making it a valid argument to `dist`_. The former sounds like a method invocation.